### PR TITLE
LPS-180895 | Automation for FDSViews#CanUpdateNumberOfViews + FDSViews#CanAssertPagination

### PR DESCRIPTION
**Pull request by alexmds2004**
**Description**

- **Given** access to the Datasets admin page
- **When** The user goes to add a new Dataset
- **And** Add a new view
- **And** the user deletes a View created
- **And** back to the Datasets admin page
- **Then** confirm that the views column shows 0 views

**Jira ticket**
https://issues.liferay.com/browse/LPS-180895

--- 

**Description**

- **Given** access to the Datasets admin page 
- **When** The user goes to add a new Dataset
- **And** Add a 6 views
- **And** Change the pagination to 4 items
- **Then** Confirm that 2 pages are shown
- **When** the user deletes two Views
- **Then** Confirm that only one page is displayed

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-180899

### CODE REVIEW

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontedn

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path [hunk 1]
```diff
@@ -35,11 +35,39 @@
...
+</tr>
+<tr>
+	<td>DATA_SET_VIEW_ENTRY_NAME</td>
+	<td>//li[contains(@class,'list-group-item')]//div[contains(@class,'justify-content-center')]//p[(@class="list-group-title") and contains (.,'${entryName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIEWS_NUMBER</td>
+	<td>//div[@class="dnd-td"] [3]</td>
+	<td></td>
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path:69]: Could you change this path to a path more specific one? Something that can receive the value of the amount of views to confirm later

Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path:69]: oops, it looks like NUMBER_OF_VIEWS does that, you can use it and delete VIEWS_NUMBER

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		task ("And the user deletes a View created") {
+			FrontendDataSetAdmin.deleteDataSetView(viewName = "View Test");
+		}
+
+		task ("And back to the Datasets admin page") {
+			Navigator.gotoBack();
+		}
+
+		task ("Then confirm that the views column shows 0 views") {
+			AssertElementPresent(locator1 = "FrontendDataSetAdmin#VIEWS_NUMBER");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase:142]:
```suggestion
			AssertElementPresent(
				locator1 = "FrontendDataSetAdmin#NUMBER_OF_VIEWS",
				numberOfViews = 0);
```

This step the way it is done does not confirm that the number is zero, only that the number is present.

**Review by mateusralv — CHANGES_REQUESTED** [review]
Hi @alexmds2004 I added a step to the test, could you take a look at it? thanks! 😃

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
Hey @alexmds2004 Could you update this PR?

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Review by mateusralv — APPROVED** [review]

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**